tests/test_node.cpp

```diff
@@ -57,7 +57,7 @@ BOOST_FIXTURE_TEST_CASE(test_node_validate_id, FixtureInitClose)
     error = EN_addnode(ph, (char *)"N 3", EN_JUNCTION, &index);
     BOOST_REQUIRE(error == 252);
 
-    error = EN_addnode(ph, (char *)"N\"3", EN_JUNCTION, &index);
+    error = EN_addnode(ph, (char *)"\"N3", EN_JUNCTION, &index);
     BOOST_REQUIRE(error == 252);
```

Comment:
with matlab test: 
![image](https://user-images.githubusercontent.com/2945956/56419934-bad21700-62a4-11e9-955b-474c5d0d1729.png)

Returns of this function with this node ID is zero, not 252.

Reply:
In C/C++ code, a double quote in a string literal needs to be preceeded by an escape character (the backslash) so it is not interpreted as the end of string marker. So in the C code for _test_node.cpp_, the literal `"\"N3"` is read as `"N3` by the compiler and fails to be a valid ID name. In Matlab, I'm guessing that `'\"N3' `is interpreted as `\"N3` which is valid since it doesn't begin with a double quote.

Reply:
@LRossman , Thanks! Then this PR looks ok.